spec/requests/pages/questions_controller_spec.rb

```diff
@@ -22,7 +22,7 @@
     login_as_editor_user
   end
 
-  describe "#new" do
+  describe "#create" do
```

Comment:
Good spot on this being misnamed. Feels like there should be a test for `#new` in here as well, but that doesn't have to be part of this PR.

Reply:
This is fairly large already. I've added a tech debt ticket here https://trello.com/c/l7OUqJEN/1223-tech-debt-add-missing-request-spec-for-questioncontroller-new-action